prow/tide/tide_test.go

```diff
@@ -1406,3 +1421,201 @@ func TestSync(t *testing.T) {
 		}
 	}
 }
+
+func TestIsPassing(t *testing.T) {
+	headSHA := "head"
+	success := "SUCCESS"
+	failure := "FAILED"
+	testCases := []struct {
+		name                  string
+		passing               bool
+		useRequiredChecksOnly bool
+		combinedContexts      map[string]string
+		requiredContexts      []string
+	}{
+		{
+			name:                  "required checks but not used - passing",
```

Comment:
`"required checks but not used"` what does this mean? Maybe `"no required checks configured"`?

Reply:
Wait, why is this a valid case? Shouldn't we never query GitHub for the list of required checks if the `useRequiredChecksOnly` field is not `true`?

Reply:
Right this is what the test is checking, I renamed the name.